### DISCUSSION TITLE
404 error on some (not all) uploaded js files

### DISCUSSION THREAD

**Discussion by Wandybrine** [community/community#22074] [Other Feature Feedback, Questions, & Ideas]
<p>I have uploaded 4 files to my repository (one html file, three js files).</p>
<p>Scripts load and render fine on my local machine. But from my github URL (<a href="https://wandybrine.github.io/module4assignment/site/" rel="nofollow noopener">https://wandybrine.github.io/module4assignment/site/</a>) I get a 404 error on two of the js files.</p>
<p><img src="https://user-images.githubusercontent.com/67928045/181068775-b21872d9-6e6f-4223-a00f-bc25bd73ed83.png" alt="Screen Shot 1" data-base62-sha1="dZ6USsg19mWa9aKsegOtshubxhO" width="655" height="195"></p>
<p>It loads the same from another computer, so I know it is not a cached problem.</p>
<p>Thanks in advance for any help!</p>

**Comment by Wandybrine** [comment]
<p>To add more context to my problem, it shows here on my project page that all 4 files have been uploaded. The problem, however, is that the files aren’t recognized when viewing my github pages html file. So, my index.html file is getting 404 not found errors on the speakhello.js and speakgoodbye.js files, but as far as I can tell, they are right where they are supposed to be. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/50dbaac4acc2ddda0eff8bc13348fd8bd66c1e59.png" data-download-href="/uploads/short-url/bxiPAMoP6gszknDEp0Oo1pBU8Hv.png?dl=1" title="Screen Shot 2" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/67928045/181068778-71e4e1af-804f-4d5f-9ff6-a6d7aeaffef2.png" alt="Screen Shot 2" data-base62-sha1="bxiPAMoP6gszknDEp0Oo1pBU8Hv" width="690" height="282" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/50dbaac4acc2ddda0eff8bc13348fd8bd66c1e59_2_690x282.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/50dbaac4acc2ddda0eff8bc13348fd8bd66c1e59_2_1035x423.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/50dbaac4acc2ddda0eff8bc13348fd8bd66c1e59.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/50dbaac4acc2ddda0eff8bc13348fd8bd66c1e59_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2</span><span class="informations">1150×471 43.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/wandybrine">@Wandybrine</a> Thank you for being here! Sometimes the letter case (uppercase/lowercase) is a source of the 404 error.  I noticed the site is looking for SpeakGoodBye.js <em>(renders a 404)</em> instead of SpeakGoodbye.js <em>(loads successfully)</em>.</p>
<p>Here is another example thread that may be helpful: <a href="https://github.community/t/my-web-site-loses-formatting-after-uploading/123911/5" class="inline-onebox">My web site loses formatting after uploading</a></p>

**Comment by Wandybrine** [comment]
<p>That worked, thanks! Is the uppercase/lowercase issue exclusive to github pages?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Great! I am glad that helped you.  This pattern is not exclusive to GitHub Pages, errors related case-sensitive filename systems <a href="https://github.community/t/github-pages-does-not-display-repositories-other-than-the-original/10247/10">do occur</a> on other platforms.</p>
